content/en/tracing/trace_collection/open_standards: update examples (#15275)
DataDog/documentation: content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

   # ...
   ```

>    In this case, we do not want to use the `hostmetrics` receiver because those metrics will be emitted by the Datadog Agent.

Reviewer: ```suggestion
   In this case, don't use the `hostmetrics` receiver because those metrics will be emitted by the Datadog Agent.
```
